modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -198,6 +198,15 @@ const areSimilarFormats = (editor: Editor, formatName: string, otherFormatName:
   });
 };
 
+const isSelectorFormat = (format: Format): format is SelectorFormat =>
+  Obj.hasNonNullableKey(format as any, 'selector');
+
+const isBlockFormat = (format: Format): format is BlockFormat =>
+  Obj.hasNonNullableKey(format as any, 'block');
+
+const isInlineFormat = (format: Format): format is InlineFormat =>
+  Obj.hasNonNullableKey(format as any, 'block');
```

Comment:
That doesn't look right — `InlineFormat` doesn't have a 'block' field

Reply:
Maybe check `!isSelectorFormat(format) && Obj.hasNonNullableKey(format as any, 'inline')`?

Reply:
See previous comments on this.